src/Components/OrgCreationForm.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { CreateOrgDetails } from '../utilities/Types';
+import  formatPhoneNumber  from '../utilities/formatPhoneNumber'
+import apiCalls from '../utilities/apiCalls';
+
+const OrgCreationForm: React.FC<{}> = () => {
```

Comment:
I think we agreed this React.FC for typescript was deprecated and can be removed